src/discussions/discussions-home/BlackoutInformationBanner.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react';
+
+import { useSelector } from 'react-redux';
+
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { PageBanner } from '@edx/paragon';
+
+import { selectBlackoutDate } from '../data/selectors';
+import messages from '../messages';
+import { inBlackoutDateRange } from '../utils';
+
+function BlackoutInformationBanner({
+  intl,
+}) {
+  const isDiscussionsBlackout = inBlackoutDateRange(useSelector(selectBlackoutDate));
+  const [showBanner, setShowBanner] = useState(true);
+
+  return (
+    <PageBanner
+      variant="warning"
+      show={isDiscussionsBlackout && showBanner}
+      dismissible
+      onDismiss={() => setShowBanner(false)}
+    >
+      <div style={{ fontWeight: '500' }}>
```

Comment:
you can use `font-weight-500` instead of inline style.

Reply:
fixed it, thanks